language/tools/move-cli/README.md

```diff
@@ -130,33 +194,126 @@ fun main(account: &signer) {
 }
 ```
 
-This script invokes the `publish` function of our `Test` module, which will publish a resource of type `Test::Resource` under the signer's account:
+This script invokes the `publish` function of our `Test` module, which will
+publish a resource of type `Test::Resource` under the signer's account:
 
 ```
-λ move run script.move --signers 0xf -c
-Compiling 1 user module(s)
+λ move run test_script.move --signers 0xf -v
+Compiling transaction script...
 Changed resource(s) under 1 address(es):
-  Changed 1 resource(s) under address 0000000000000000000000000000000f:
-    Added type 00000000::Test::Resource: [U64(10)]
-Committed changes.
+  Changed 1 resource(s) under address 0000000000000000000000000000000F:
+      Added type 0x2::Test::Resource: [U64(10)]
 ```
 
 We can also inspect this newly published resource using `move view`:
 
 ```
-λ move view storage/0x0000000000000000000000000000000f/resources/0x00000000000000000000000000000002\:\:Test\:\:Resource
-resource 00000000::Test::Resource {
-    i: 10
+λ move view storage/0x0000000000000000000000000000000F/resources/0x00000000000000000000000000000002::Test::Resource.lcs
+resource 0x2::Test::Resource {
+        i: 10
 }
 ```
 
+### Cleaning state
 
-## Using the CLI with Libra modules and genesis state
+Since state persists from one call to the Move CLI to another, there will
+frequently be times where you want to start again at a clean state.  This can
+be done using the `move clean` command. This command will remove all previously
+published modules and resources from the state, and will start you back at a
+fresh state empty state.
```

Comment:
Might be easier to just say: this deletes the contents of the `storage` directory (and whatever else it does).